src/api/spec/features/beta/webui/requests_index_spec.rb

```diff
@@ -0,0 +1,76 @@
+require 'browser_helper'
+
+RSpec.describe 'Requests Index' do
+  let(:submitter) { create(:confirmed_user, :with_home, login: 'kugelblitz') }
+  let(:receiver) { create(:confirmed_user, :with_home, login: 'titan') }
+  let(:target_project) { receiver.home_project }
+  let(:target_package) { create(:package, name: 'goal', project_id: target_project.id) }
+  let(:source_project) { submitter.home_project }
+  let(:source_package) { create(:package, name: 'ball', project_id: source_project.id) }
```

Comment:
```suggestion
  let(:source_package) { create(:package, name: 'ball', project: source_project) }
```

Reply:
Same here, the suggestion was not applied